tests/acceptance/30_generic_methods/augeas_commands.cf

```diff
@@ -0,0 +1,66 @@
+ #######################################################
+#
+# Read a file into a string
+#
+#######################################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("${this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+    "tmp"          string => getenv("TEMP", 1024); 
+    "commands_1"   string => "print /augeas/load/";
+    "commands_2"   string => "get /files/etc/hosts/1/ipaddr";
+    "noautoload_1" string => "true";
+    "noautoload_2" string => "false";
+}
+#######################################################
+
+bundle agent test
+{
+  methods:
+    "ph1" usebundle => augeas_commands("prefix", "var1", "${init.commands_1}", "${init.noautoload_1}");
+    "ph2" usebundle => augeas_commands("prefix", "var2", "${init.commands_2}", "${init.noautoload_2}");
+    "ph3" usebundle => augeas_commands("prefix", "var3", "${init.commands_2}", "");
+}
+
+#######################################################
+
+bundle agent check
+{
+  classes:
+    
+    "ok_1" expression => "augeas_commands_var1_kept.!augeas_commands_var1_repaired.!augeas_commands_var1_error";
+    "ok_2" expression => "augeas_commands_var2_kept.!augeas_commands_var2_repaired.!augeas_commands_var2_error";
+    "ok_3" expression => "augeas_commands_var3_kept.!augeas_commands_var3_repaired.!augeas_commands_var3_error";
+    "ok_var1" expression => isvariable("prefix.var1");
+    "ok_var2" expression => isvariable("prefix.var2");
+    "ok_var3" expression => isvariable("prefix.var3");
+    "ok_var1_cont" expression => strcmp("${prefix.var1}","/augeas/load");
+    "ok_var2_cont" expression => strcmp("${prefix.var2}","/files/etc/hosts/1/ipaddr = 127.0.0.1");
```

Comment:
this may not always be true on the test system. Getting the uid of the root user like in the other test is probably a better idea

Reply:
okay i change this

Reply:
https://github.com/Normation/ncf/pull/1178/commits/985b88cd0516595650190de0723e6e18b1cd699e